sormas-ui/src/main/java/de/symeda/sormas/ui/externalmessage/processing/ExternalMessageProcessingUIHelper.java

```diff
@@ -219,18 +229,37 @@ private static CommitDiscardWrapperComponent<SampleEditForm> getSampleEditCompon
 			sample.isInJurisdiction(),
 			sampleController.getDiseaseOf(sample),
 			false);
+		sampleEditComponent.getWrappedComponent().setHeading(I18nProperties.getString(Strings.headingExternalMessageSampleInformation));
 
 		// add existing tests to edit component
 		int caseSampleCount = sampleController.caseSampleCountOf(sample);
 
 		List<PathogenTestDto> existingTests = FacadeProvider.getPathogenTestFacade().getAllBySample(sample.toReference());
-		for (PathogenTestDto existingTest : existingTests) {
-			PathogenTestForm pathogenTestForm = sampleController.addPathogenTestComponent(sampleEditComponent, existingTest, caseSampleCount, false);
+
+		Label existingTestsSeparator = new Label("<br/><hr/><br/>", ContentMode.HTML);
+		existingTestsSeparator.setWidth(100f, Sizeable.Unit.PERCENTAGE);
+		existingTestsSeparator.setVisible(!existingTests.isEmpty());
+		sampleEditComponent.addComponent(existingTestsSeparator, sampleEditComponent.getComponentCount() - 1);
+
+		Label existingTestsLabel = new Label(I18nProperties.getString(Strings.headingExternalMessageExistingPathogenTests));
+		existingTestsLabel.addStyleName(H3);
+		existingTestsLabel.setVisible(!existingTests.isEmpty());
+		sampleEditComponent.addComponent(existingTestsLabel, sampleEditComponent.getComponentCount() - 1);
```

Comment:
The above 3 lines could be extracted to a method to get the configured existingTestsLabel . The extracted method could be reused in this class for newTestsLabel